test/java/seedu/addressbook/commands/SortCommandTest.java

```diff
@@ -0,0 +1,34 @@
+package seedu.addressbook.commands;
+
+import org.junit.Test;
+import seedu.addressbook.data.AddressBook;
+import seedu.addressbook.data.person.ReadOnlyPerson;
+import seedu.addressbook.util.TestUtil;
+import seedu.addressbook.util.TypicalPersons;
+
+import java.util.List;
+
+import static org.junit.Assert.assertTrue;
+
+public class SortCommandTest {
+
+    @Test
+    public void execute() throws Exception {
+
+        TypicalPersons td = new TypicalPersons();
+        AddressBook addressBook = TestUtil.createAddressBook(td.dan, td.candy, td.bill, td.amy);
+        List<ReadOnlyPerson> original = addressBook.getAllPersons().immutableListView();
+
+        SortCommand command = new SortCommand();
+        command.setData(addressBook, original);
+        CommandResult result = command.execute();
+
+        assertTrue(result.getRelevantPersons().isPresent());
+
+        List<? extends ReadOnlyPerson> actual = result.getRelevantPersons().get();
+        List<? extends ReadOnlyPerson> expected = TestUtil.createList(td.amy, td.bill, td.candy, td.dan);
+
+        assertTrue(actual.equals(expected));
```

Comment:
Is it intentional that you aren't checking the resultant `feedbackToUser` here? Since the precise output is checked in the I/O test.

Reply:
Good point! Yes I've already checked it in the I/O tests so there is no need to duplicate the test.